proton.in

```diff
@@ -332,7 +332,10 @@ for dll in dlloverrides:
         s = s + ";" + dll + "=" + setting
     else:
         s = dll + "=" + setting
-env["WINEDLLOVERRIDES"] = s
+if 'WINEDLLOVERRIDES' in os.environ:
```

Comment:
All external env variables recognized by Proton have `PROTON_` prefix. Maybe use `PROTON_WINEDLLOVERRIDES`? Also, documentation for Runtime config options in `README.md` should probably be updated to reflect your change.